jdaviz/configs/imviz/plugins/coords_info/coords_info.py

```diff
@@ -460,9 +461,15 @@ def _image_viewer_update(self, viewer, x, y):
             self.row3_unreliable = False
 
         maxsize = int(np.ceil(np.log10(np.max(active_layer.layer.shape)))) + 3
-        fmt = 'x={0:0' + str(maxsize) + '.1f} y={1:0' + str(maxsize) + '.1f}'
+        if any(['nan' in map(str, (x, y))]):
```

Comment:
I think NaN could also result from invalid FITS WCS conversion but I am not 100% sure now. No reason to remove unless somehow this also deteriorates the performance significantly?

Reply:
I agree that this doesn't affect performance. 

On the other hand, is it meaningful or useful to display "nan" as a pixel coordinate?

Reply:
I think it just crashes if `np.nan` is not handled. I didn't add this logic for fun but the exact details, I can't quite recall now. I just know it was needed.